### PR TITLE
Update 07-tracing.md

### PR DESCRIPTION
**Pull request by hanshal101**
Fixes #<XXXX>.

### Motivation

*Explain the content here.*
*Explain why you want to make the changes and what problem you're trying to solve.*

### Modifications

*Describe the modifications you've done.*

### Documentation

- Does this pull request introduce a new feature? (yes / no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
- If a feature is not applicable for documentation, explain why?
- If a feature is not documented yet in this PR, please create a followup issue for adding the documentation

### CODE REVIEW

File: docs/zh/design-document/07-tracing.md [hunk 1]
```diff
@@ -1,87 +1,128 @@
...
+    . Jiànzào ();
+dāng shǐyòng “EventMeshHTTPServer” lèi de “init()” fāngfǎ shí,“AbstractHTTPServer” lèi jiāng huòqǔ gēnzōng qì
+super.OpenTelemetryTraceFactory = new OpenTelemetryTraceFactory(eventMeshHttpConfiguration);
+super.Tracer = openTelemetryTraceFactory.GetTracer(this.GetClass().ToString());
+super.TextMapPropagator = openTelemetryTraceFactory.GetTextMapPropagator();
+nàme “AbstractHTTPServer” lèi zhōng de gēnzōng jiāng qǐ zuòyòng.
+Wèntí
+rúhé zài “OpenTelemetryTraceFactory” lèi zhōng shèzhì bùtóng de dǎochū qì? (Jiějuéle)
+cóng shǔxìng zhòng huòqǔ dǎochū qì lèixíng hòu, rúhé chǔlǐ tā.
+
```
Comment (mxsm) [inline docs/zh/design-document/07-tracing.md:100]: There are some problems here. Please check it

**Review by mxsm — COMMENTED** [review]
@hanshal101 First  thanks for you contribution. You can create issue first and then link this pr to it. Translation result of document has some problems please fix it.

**Comment by hanshal101** [comment]
Can you please assign me this task it would be great. I'll try to complete the remaining tasks soon.

File: docs/zh/design-document/07-tracing.md [hunk 2]
```diff
@@ -1,87 +1,128 @@
...
-        .setScheduleDelay(5, TimeUnit.SECONDS) // set time between two different exports
-        .build();
+     BatchSpanProcessor.builder（出口商）
+         .setMaxExportBatchSize(512) // 设置要使用的最大批量大小
+         .setMaxQueueSize(2048) // 设置队列大小。 这必须 >= 导出批量大小
+         .setExporterTimeout(
+             30, TimeUnit.SECONDS) // 设置导出在获取之前可以运行的最长时间
+         // 中断
+         .setScheduleDelay(5, TimeUnit.SECONDS) // 设置两次不同导出之间的时间
+         。建造（）;
```
Comment (Pil0tXia) [inline docs/zh/design-document/07-tracing.md:23]: Please notice code block translation mistake.

File: docs/zh/design-document/07-tracing.md [hunk 3]
```diff
@@ -1,87 +1,128 @@
...
 BatchSpanProcessor batchSpansProcessor =
-    BatchSpanProcessor.builder(exporter)
-        .setMaxExportBatchSize(512) // set the maximum batch size to use
-        .setMaxQueueSize(2048) // set the queue size. This must be >= the export batch size
-        .setExporterTimeout(
-            30, TimeUnit.SECONDS) // set the max amount of time an export can run before getting
-        // interrupted
-        .setScheduleDelay(5, TimeUnit.SECONDS) // set time between two different exports
-        .build();
+     BatchSpanProcessor.builder（出口商）
```
Comment (Pil0tXia) [inline docs/zh/design-document/07-tracing.md:16]: Code Block translation mistake.

**Review by Pil0tXia — COMMENTED** [review]
Thanks for your contribution. However, please ponder thoroughly before submitting a merge request, ensuring the format of the Markdown document is accurate and provides correct translation. It is quite challenging to extract valuable information from your commits.

File: docs/zh/design-document/07-tracing.md (hunk 3 above)
Comment (hanshal101) [reply docs/zh/design-document/07-tracing.md:16]: Can you please elobrate this issue.

File: docs/zh/design-document/07-tracing.md [hunk 4]
```diff
@@ -1,87 +1,128 @@
...
+         .setMaxExportBatchSize(512) // 设置要使用的最大批量大小
+         .setMaxQueueSize(2048) // 设置队列大小。 这必须 >= 导出批量大小
+         .setExporterTimeout(
+             30, TimeUnit.SECONDS) // 设置导出在获取之前可以运行的最长时间
+         // 中断
+         .setScheduleDelay(5, TimeUnit.SECONDS) // 设置两次不同导出之间的时间
+         。build（）;
```
Comment (hanshal101) [inline docs/zh/design-document/07-tracing.md:23]: I have made changes on the 23 line please review it

File: docs/zh/design-document/07-tracing.md (hunk 2 above)
Comment (hanshal101) [reply docs/zh/design-document/07-tracing.md]: I have made changes please review

**Review by hanshal101 — COMMENTED** [review]
Heyy, @Pil0tXia I have made all the necessary changes please review them and let me know if more things are need to be updated

File: docs/zh/design-document/07-tracing.md [hunk 5]
```diff
@@ -1,87 +1,128 @@
...
 BatchSpanProcessor batchSpansProcessor =
-    BatchSpanProcessor.builder(exporter)
-        .setMaxExportBatchSize(512) // set the maximum batch size to use
-        .setMaxQueueSize(2048) // set the queue size. This must be >= the export batch size
-        .setExporterTimeout(
-            30, TimeUnit.SECONDS) // set the max amount of time an export can run before getting
-        // interrupted
-        .setScheduleDelay(5, TimeUnit.SECONDS) // set time between two different exports
-        .build();
+     BatchSpanProcessor.builder（exporter）
```
Comment (hanshal101) [inline docs/zh/design-document/07-tracing.md:16]: Made changes in line 16

**Comment by hanshal101** [comment]
I'm really sorry I couldn't make the PR better as I'm a newbie for open-source contributions. Please guide me through on making a good PR. It would be really helpful.

**Comment by Pil0tXia** [comment]
Allow me to be candid and honest with you. It is not appropriate nor advisable to directly use machine translation on Markdown text. This not only disrupts the formatting of the document but also produces inaccurate translation results, which is irresponsible behavior within the community. For instance, characters such as `（` and `(`, `。` and `.` are distinct and should be treated differently. Additionally, there are issues of Chinese Pinyin, as pointed out by @mxsm.

I appreciate your contributions here, and you are always welcome to contribute. However, I suggest that you first familiarize yourself with the target language before engaging in translation work. This will be more helpful for you in the long run.

**Comment by hanshal101** [comment]
Thank You @Pil0tXia for the guidance. From now onwards I'll make sure to get familiar with a language and then start contributing. This was really helpful. 

Thanks once gain for helping.

**Comment by hanshal101** [comment]
For now should I close the PR

**Comment by Pil0tXia** [comment]
The EventMesh documentation has been extracted to the [eventmesh-site](https://github.com/apache/eventmesh-site) repository, where you can continue when you are ready.

**Comment by hanshal101** [comment]
Ok Thanks once again

**Pull request closed without merging**